gradle/build.go

```diff
@@ -96,9 +97,9 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		command = filepath.Join(context.Layers.Path, d.Name(), "bin", "gradle")
 	} else if err != nil {
 		return libcnb.BuildResult{}, fmt.Errorf("unable to stat %s\n%w", command, err)
-	} else {
+	} else if fi.Mode() != fs.FileMode(0755) {
```

Comment:
```suggestion
	} else {
```

It's unnecessary to check this. We can run `os.Chmod` every time. In most cases, it'll functionally be a no-op. If it fails, we're just logging a warning.

Reply:
Done